docs/Classes.html

```diff
@@ -22,7 +22,7 @@
         <a class="header-link" href="index.html">
           KituraContracts Docs
         </a>
-         (100% documented)
+         (98% documented)
```

Comment:
:cry:

Reply:
Looks like the [Operation protocol](https://github.com/IBM-Swift/KituraContracts/blob/1.0.0/Sources/KituraContracts/Contracts.swift#L1097) is missing documentation on its functions - could we add these as part of this PR?

Reply:
done